noir-projects/noir-protocol-circuits/crates/rollup-lib/src/abis/block_root_rollup_data.nr

```diff
@@ -23,17 +25,29 @@ pub struct BlockRootRollupData {
     // Its hash is checked against the previous archive.
     pub previous_block_header: BlockHeader,
 
+    // <-- NOTE: start_blob_accumulator and  final_blob_challenges exist here and not BlockRootRollupBlobData
+    // so we can inject easily into EmptyBlockRootRollupInputs:
+
+    // The current blob accumulation state across the epoch.
+    // Injected here and checked vs the previous end state in each block merge and root rollup circuit:
+    pub start_blob_accumulator: BlobAccumulatorPublicInputs,
+    // Finalized challenges z and gamma for performing blob batching. Shared value across the epoch
+    // Precompjuted, injected here, and confirmed in the root rollup circuit:
+    pub final_blob_challenges: FinalBlobBatchingChallenges,
+
+    // -->
+
     // TODO(#7346): Temporarily added prover_id while we verify block-root proofs on L1
     pub prover_id: Field,
 }
 
 pub struct BlockRootRollupBlobData {
     // Fields for blob verification made of all tx effects in this block
-    // (will likely change to be accumulated and sent up to the final root)
     pub blobs_fields: [Field; FIELDS_PER_BLOB * BLOBS_PER_BLOCK],
-    // see blob/src/blob.nr -> evaluate_blob() -  these are used for creating the challenge z
-    pub blob_commitments: [BlobCommitment; BLOBS_PER_BLOCK],
+    // see blob/src/blob.nr -> evaluate_blob() -  these are used for creating each blob's challenge z_i
+    pub blob_commitments: [BLSPoint; BLOBS_PER_BLOCK],
     // Flat sha256 hash of the EVM blob hashes, can be injected here as the contract checks its validity vs the blob_public_inputs
     // NB: to fit it into a field, we truncate to 31 bytes
+    // TODO(MW): Replace with v?
```

Comment:
`blobs_hash` seems nice and readable, so maybe no need to replace with `v`

Reply:
Hahahaha

<img width="588" alt="image" src="https://github.com/user-attachments/assets/4e263292-24ad-43d4-9687-b7cf874cb1b5" />